src/main/java/com/codereview/ipcounter/handler/IpCounterEventHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.codereview.ipcounter.handler;
+
+import com.codereview.ipcounter.service.ArgumentProcessor;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class IpCounterEventHandler {
```

Comment:
I don't understand why you need this class. I would understand seeing it in tests.